Dockerfile

```diff
@@ -37,8 +37,7 @@ ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER=/x86_64-linux-musl-native/bin/
 # builder stage
 WORKDIR /iroha
 COPY . .
-# FIXME: consider building only `irohad`, `iroha`, and `kagami`?
-RUN cargo build --target x86_64-unknown-linux-musl --profile deploy
+RUN cargo build -p irohad --target x86_64-unknown-linux-musl --profile deploy
```

Comment:
I think kagami should also be build, #4673 relies on it:
https://github.com/hyperledger/iroha/blob/730ca759395e47f2e63d6f35ab6c447007fb04a1/configs/swarm/docker-compose.single.yml#L25